src/components/common/EntityInfo.vue

```diff
@@ -40,29 +54,32 @@
 <script lang="ts">
 // Libraries
 import { Component, Vue } from 'vue-property-decorator'
-import { Getter } from 'vuex-class'
+import { Getter, State } from 'vuex-class'
 
 // Interfaces
-import { GetterIF } from '@/interfaces'
+import { BusinessContactIF, BusinessInformationIF, GetterIF } from '@/interfaces'
 
 @Component({})
 export default class EntityInfo extends Vue {
+  // Global Store
+  @State(state => state.stateModel.businessInformation)
+  readonly businessInformation!: BusinessInformationIF
+
+  @State(state => state.stateModel.defineCompanyStep.businessContact)
+  readonly businessContact!: BusinessContactIF
+
   // Global getters
   @Getter isEntityType!: GetterIF
   @Getter isTypeBcomp!: GetterIF
   @Getter isTypeCoop!: GetterIF
-  @Getter getNameRequestNumber!: GetterIF
   @Getter getTempId!: GetterIF
   @Getter getApprovedName!: GetterIF
 
   /** The entity title  */
   private entityTitle (): string {
     if (this.isTypeBcomp) {
-      return 'BC Benefit Company Incorporation Application'
-    } else if (this.isTypeCoop) {
-      return 'Incorporate a BC Cooperative Association'
+      return 'Correction - Incorporation Application'
```

Comment:
This could use an update once we see the designs for alterations / profile view etc. Then can populate the entity info accordingly. This goes for the Breadcrumb as well.